### ISSUE TITLE
cannot read property `secretKey` of undefined account

### ISSUE THREAD

**Issue by nmushegian** [NexusDevelopment/dapple#75]
Look at NexusDevelopment/dappsys @ 35888f38f4be4ad933eede72264da3616136a981

```
/Users/nikolai/devenv/dapple/node_modules/solc/bin/soljson-latest.js:1
(function (exports, require, module, __filename, __dirname) { var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=typeof window==="object";var ENVIRONMENT_IS_WORKER=typeof importScripts==="function";var ENVIRONMENT_IS_NODE=typeof process==="object"&&typeof require==="function"&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;var ENVIRONMENT_IS_SHELL=!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_NODE&&!ENVIRONMENT_IS_WORKER;if(ENVIRONMENT_IS_NODE){if(!Module["print"])Module["print"]=function print(x){process["stdout"].write(x+"\n")};if(!Module["printErr"])Module["printErr"]=function printErr(x){process["stderr"].write(x+"\n")};var nodeFS=require("fs");var nodePath=require("path");Module["read"]=function read(filename,binary){filename=nodePath["normalize"](filename);var ret=nodeFS["readFileSync"](f

TypeError: Cannot read property 'secretKey' of undefined
    at Blockchain.processCall (/Users/nikolai/devenv/dapple/node_modules/ethersim/lib/blockchain.js:495:50)
    at Blockchain.processNextAction (/Users/nikolai/devenv/dapple/node_modules/ethersim/lib/blockchain.js:382:10)
    at intermediary (/Users/nikolai/devenv/dapple/node_modules/ethersim/lib/blockchain.js:370:12)
    at /Users/nikolai/devenv/dapple/node_modules/ethersim/lib/blockchain.js:415:9
    at parseBlockResults (/Users/nikolai/devenv/dapple/node_modules/ethereumjs-vm/lib/runBlock.js:142:7)
    at /Users/nikolai/devenv/dapple/node_modules/async/lib/async.js:721:13
    at /Users/nikolai/devenv/dapple/node_modules/async/lib/async.js:52:16
    at /Users/nikolai/devenv/dapple/node_modules/async/lib/async.js:264:21
    at /Users/nikolai/devenv/dapple/node_modules/async/lib/async.js:44:16
    at /Users/nikolai/devenv/dapple/node_modules/async/lib/async.js:718:17
```

**Comment by ryepdx** [comment]
I can't duplicate this error: Dappsys @ 35888f38f4be4ad933eede72264da3616136a981 fails due to incorrect imports. @nmushegian, can you verify that this is still broken for you after #85 gets merged in? If it is, I'll go ahead and downgrade my copy of `solc` to see if I can't figure out what's going on.

**Comment by ryepdx** [comment]
Just triggered it myself in the course of writing a token contract. Will investigate further.

**Comment by ryepdx** [comment]
Didn't get much done on this today, but it likes like it's the result of Ethersim not handling thrown exceptions in the EVM very gracefully. I can reliably reproduce this using the `throw` keyword in Solidity.

**Comment by ryepdx** [comment]
Addressed by #95
